config/routes.rb

```diff
@@ -36,4 +36,14 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  post "/applications", to: "applications#create"
+  get "/applications/new", to: "applications#new"
+  patch "/applications/:id/pets/:pet_id", to: "application_pets#update"
+  get "/applications/:id", to: "applications#show"
+  patch "/applications/:id", to: "applications#update"
+
+  get "/admin/shelters", to: "admin/shelters#index"
+  get "/admin/applications/:id", to: "admin/applications#show"
+  patch "/admin/applications/admin/applications/:id", to: "admin/applications#update" #why does this need to be doubled up?
```

Comment:
There isn't a reason `admin/applications` should be in there twice. `patch "/admin/applicaitons/:id"` should suffice.